cpp/monitoring/gauge.h

```diff
@@ -25,12 +25,13 @@ class Gauge : public Metric {
 
   void Set(const LabelTypes&... labels, double value);
 
+  // TODO(alcutter): Not over the moon about having this here.
```

Comment:
Nice TODO.  Do you have any idea what might be done about it?  The TODO would be more informative if you can write what steps are needed to remove it later.